### PR TITLE
Static pages

### PR DESCRIPTION
**Pull request by langtonfavor**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
+  get 'static_pages/contact'
+  root 'static_pages#home'
```
Comment (JOlsen-Neslo) [inline config/routes.rb]: The root route must be first, all other must follow

**Review by JOlsen-Neslo — CHANGES_REQUESTED** [review]

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
+  get 'users/new'
+  root             'static_pages#home'
```
Comment (JOlsen-Neslo) [inline config/routes.rb:2]: The root route is still not at the beginning

**Review by JOlsen-Neslo — CHANGES_REQUESTED** [review]

File: config/routes.rb [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+  get 'help'    => 'static_pages#help'
+  get 'about'   => 'static_pages#about'
+  get 'contact' => 'static_pages#contact'
+  get 'static_pages/contact'
```
Comment (JOlsen-Neslo) [inline config/routes.rb]: Why are both these here?

**Review by JOlsen-Neslo — CHANGES_REQUESTED** [review]

**Review by JOlsen-Neslo — APPROVED** [review]

**Comment by JOlsen-Neslo** [comment]
Don't think I have permissions to merge

**Comment by langtonfavor** [comment]
will  merge it

**Pull request merged**